index.js

```diff
@@ -147,6 +156,10 @@ FacebookInsightStream.prototype._initItem = function ( item ) {
             }
             return result
         })
+        .catch( function ( error ) {
```

Comment:
@liorrozen @alonweissfeld I have added the support to the _initItem method as well